### DISCUSSION TITLE
Gather End User Feedback

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#33] [Project Proposal]
Hello there!

We have discussed in the past about raising awareness of environmental sustainability as well as gathering insights from the community. I believe the best way is to craft and send out a questionnaire to the community. Questionnaires help to raise awareness while asking people to create some thoughts about it.

The questionnaire should collect three kinds of information:
- Demographic input
  - the size of the company
  - end user, vendor, consulting etc.
  - industry
- Technology Standpoint
  - Which tools do they use to monitor resource utilization?
  - Which metrics do they try to gather?
  - Are they consider sustainability within their architecture?
  - Which tools/methods do they use to optimize the workload?
  - Which tools/methods do they use to schedule and scale their workload?
  - Have they implemented any approach to measure power consumption?
- Sustainability Responsibilities
  - Are they partners or actively working with other foundations on sustainability?
  - Have their company sustainability targets?
  - Does the company has other activities to improve sustainability?

I think, I for sure miss some questions - wdyt? 

Out of the results, we can craft a report and maybe provide further input for our future work as well as valuable feedback on some existing tooling.

**Comment by catblade** [comment]
Hello Max,

When you wrote “the community” I had first thought you meant participants in the WG, but from reading further, it seems it is more targeted to a specific demographic, ie companies.

Perhaps we can include results from such a survey into the landscape doc deliverable under the use cases section.  I would expand into asking questions on challenges and perhaps focus on that, because often technology is not adopted because of challenges.  (How are they attempting to gather metrics?  Do they have some sort of certification process?  How are they taking the measurements?  What are blockers currently to getting the solution you want?  Do they have faith in the tooling?  What are they currently exploring?)  I do think that may be a useful thing to discuss at a working meeting, as different groups/companies/individuals may be interested in specific information to inform their efforts.

We may also want to include another survey for various foundations so we know what each is doing, what their focus is, and how large their efforts currently are.

Best wishes,
--Marlow


From: Max Körbächer ***@***.***>
Sent: Saturday, August 27, 2022 5:13 AM
To: cncf/wg-env-sustainability ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [cncf/wg-env-sustainability] Gather End User Feedback (Discussion #33)


Hello there!

We have discussed in the past about raising awareness of environmental sustainability as well as gathering insights from the community. I believe the best way is to craft and send out a questionnaire to the community. Questionnaires help to raise awareness while asking people to create some thoughts about it.

The questionnaire should collect three kinds of information:

  *   Demographic input
     *   the size of the company
     *   end user, vendor, consulting etc.
     *   industry
  *   Technology Standpoint
     *   Which tools do they use to monitor resource utilization?
     *   Which metrics do they try to gather?
     *   Are they consider sustainability within their architecture?
     *   Which tools/methods do they use to optimize the workload?
     *   Which tools/methods do they use to schedule and scale their workload?
     *   Have they implemented any approach to measure power consumption?
  *   Sustainability Responsibilities
     *   Are they partners or actively working with other foundations on sustainability?
     *   Have their company sustainability targets?
     *   Does the company has other activities to improve sustainability?

I think, I for sure miss some questions - wdyt?

Out of the results, we can craft a report and maybe provide further input for our future work as well as valuable feedback on some existing tooling.

**Comment by mkorbi** [comment]
Surveys can be requested via the CNCF service desk: https://www.cncf.io/services-for-projects/#marketing - search for Survey
Survey data from past surveys can be found here: https://github.com/cncf/surveys

**Comment by mkorbi** [comment]
Please find here the working document https://docs.google.com/document/d/1_yTOtWeosDsGpSy--gzIlous_zxGcHYRXFhcg1PMCSA/edit?usp=sharing

**Comment by leonardpahlke** [comment]
Just as a follow-up here… The survey has been discussed and created 🎉 Starting next week (Tuesday 12 October) the survey will be distributed among others at the Kubernetes Community Days in Munich as well as via social media and then later at Kubecon NA 🙂. When the survey closes needs to be discussed (probably shortly after Kubecon NA), as well as the data we collected after the survey closes.

**Reply by mkorbi** [reply]
I would keep it a little bit longer open, so we can keep promoting it after KubeCon and include it in like KubeWeekly newsletter, maybe a CNCF News entry or mail etc.

There are a couple of KCD that we can reach and please to promote them too:

- KCD Munich 12/13.10. (as mentioned already, I'm going to promote the survey)
- KCD Italy 17.11. - I think the best would be to ask Ciara from Mia-Platform Team
- KCD UK 22/23.11 - think we can ask Giorgia Fiscaletti (she will be in Munich too) or maybe Amanda Brock von OpenUK
- KCD Spain 15/16.11. - 
- KCD Indonesia 26/27.11. - 
- KCD Nordic 29.11. - 
- KCD Pakistan - 25/26.11. - 

While writing, the easiest would be to drop it first in the KCD organizer channel, I can do that. Second, we should create an issue with those tasks.

**Reply by leonardpahlke** [reply]
👍, I've added this to the agenda (TAG meeting 10/19/2022)

**Reply by leonardpahlke** [reply]
As discussed in the TAG meeting 10/19/2022 we will close the survey on the 25th of November 2022

**Comment by mkorbi** [comment]
The survey is under way and will be closed in end of November, going to close this discussion.
